presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2431,9 +2430,6 @@ private Optional<LocalDynamicFilter> createDynamicFilter(PhysicalOperation build
             if (node.getDynamicFilters().isEmpty()) {
                 return Optional.empty();
             }
-            if (buildSource.getPipelineExecutionStrategy() == GROUPED_EXECUTION) {
```

Comment:
Why is this removed?

Reply:
not sure why it's removed on the git, my local branch has this code piece, I will rebase to master and push the commit again